src/main/java/no/nav/pto/veilarbportefolje/admin/AdminController.java

```diff
@@ -183,6 +183,20 @@ public String lastInnPDLBrukerData() {
         return "ferdig";
     }
 
+    @PutMapping("/pdl/lastInnDataFraPdl")
+    public String lastInnPDLBrukerData(@RequestParam String fnr) {
+        sjekkTilgangTilAdmin();
+        String aktorId = aktorClient.hentAktorId(Fnr.ofValidFnr(fnr)).get();
+        try {
+            pdlService.hentOgLagrePdlData(AktorId.of(aktorId));
+        } catch (Exception e) {
+            secureLog.info("pdl brukerdata: feil under innlastning av pdl data på bruker: {}", aktorId, e);
+        }
+        log.info("pdl brukerdata: ferdig med innlastning");
+        return "ferdig";
```

Comment:
Is it necessary to return a string? A `204 No Content` could do, I think 🤷

Reply:
I think its not necessary, just I used the same logic as when we are reading pdl data for all users.